README.md

```diff
@@ -113,35 +118,42 @@ curl http://localhost:8005/api/resources/27d469ff-9908-4b7e-a2e0-9439bb38a395/da
     }
   ],
   "links": {
+    "profile": "/api/resources/60963939-6ada-46bc-9a29-b288b16d969b/profile/",
+    "swagger": "/api/resources/60963939-6ada-46bc-9a29-b288b16d969b/swagger/",
     "next": "/api/resources/60963939-6ada-46bc-9a29-b288b16d969b/data/?page=2&page_size=1",
-    "prev": null,
-    "profile": "/api/resources/60963939-6ada-46bc-9a29-b288b16d969b/profile/"
+    "prev": null
   },
-  "meta": {"page": 1, "page_size": 1, "total": 2}
+  "meta": {"page": 1, "page_size": 20, "total": 21777}
 }
 ```
 
-This endpoint can be queried with the following operators as query string:
+This endpoint can be queried with the following operators as query string (replacing `column_name` with the name of an actual column):
 
 ```
 # sort by column
 column_name__sort=asc
 column_name__sort=desc
 
-# contains
-column_name__contains=word
-
 # exacts
 column_name__exact=word
 
+# contains (for strings only)
+column_name__contains=word
+
 # less
 column_name__less=12
 
 # greater
 column_name__greater=12
+
+# strictly less
+column_name__strictlyless=12
```

Comment:
I think we could separate words `strictly_less`?

Reply:
I wasn't sure whether we wanted to keep the "operators" without underscores, there's no issue doing it code-wise, I'll change this 👌